docs/pages/enterprise/sso.mdx

```diff
@@ -129,8 +129,11 @@ or </ScopedBlock>create a `cluster_auth_preference` resource.
 </Tabs>
 
 
-Next, define an authentication connector. Create a file called `connector.yaml`
-based on one of the following examples.
+Next, define an authentication connector. We recommend the usage of helper 
+commands to achieve this for [SAML](../setup/reference/cli.mdx#tctl-sso-configure-saml), 
```

Comment:
If we add an example of the helper commands, we should remove the links here to prevent breaking the reader's flow.